plugins/@grouparoo/marketo/src/migrations/000001-plugin-marketo-syncMode.ts

```diff
@@ -0,0 +1,23 @@
+export default {
+  up: async function (migration) {
+    await migration.sequelize.transaction(async () => {
+      // Set all existing instances to "sync" to maintain same behavior
+      await migration.sequelize.query(
```

Comment:
General question: Why does the `up` use a custom query and the `down` use `migration.bulkUpdate`?

Reply:
Because I didn't know that the `queryInterface` was a thing when I wrote the `up` part hahaha

I made it use `bulkUpdate` on both to be a bit more consistent 🤷🏽‍♂️